_posts/en/newsletters/2022-11-09-newsletter.md

```diff
@@ -0,0 +1,330 @@
+  next general update of the software.  Including the security fix
+  alongside other changes could hide that a vulnerability was being
+  fixed and reduce the chance of it being exploited).  Both Towns and
+  Osuntokun responsibly kept the vulnerability private until its fix
+  could be deployed.
+
+    Unfortunately, the second related bug was independently rediscovered by
+    someone who partnered with a miner to trigger it.  This new bug
+    affected BTCD and LND again, but it also affected at least [two
+    other][liquid and rust bitcoin vulns] significant projects or
+    services.  All users of affected systems should upgrade immediately.
+    We repeat our advice from three weeks ago for anyone using any
+    Bitcoin software sign up for security announcements from that
+    software’s development team.
+
+    With the release of this newsletter, Optech has also added a special
+    topic page where we list the names of [the amazing people who
+    responsibly disclosed a vulnerability][topic responsible
+    disclosures] that we've summarized in an Optech newsletter.  We
+    extend our most heartfelt thanks to everyone listed.  There are
+    likely several other disclosures not listed because they haven't
+    been made public yet.  We look forward to thanking those discoverers
+    and adding them to the topic page of Bitcoin security disclosure
+    heroes.  We of course also thank all the reviewers of proposals and
+    pull requests whose diligent effort prevented innumerable
+    security bugs from making it into released software.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[Relax MIN_STANDARD_TX_NONWITNESS_SIZE to 65 non-witness bytes][review club 26265]
+is a PR by instagibbs that relaxes the mempool policy's non-witness transaction size
+constraints. It allows transactions to be as small as 65 bytes, replacing
+the current policy that requires transactions to be at least 85 bytes.
+
+Since this review club meeting, this PR has been closed in favor of
+[PR 26398](https://github.com/bitcoin/bitcoin/pull/26398), which
+relaxes policy even further by disallowing _only_ 64-byte transactions.
+The relative merits of these two slightly-different policies were
+discussed during review club.
+
+{% include functions/details-list.md
+  q0="Why was the minimum transaction size 82 bytes?
+  Can you describe the attack?"
+
+  a0="The 82-byte minimum, which was introduced by PR 11423 in 2018, is
+  the size of the smallest standard payment transaction. This was
+  presented as a cleanup of the standardness rules. But in reality,
+  the change was to prevent a 64-byte transaction from being considered standard,
+  because this size allowed a
+  [spoof payment attack](https://github.com/advisories/GHSA-v55p-4chq-6grj)
```

Comment:
Suggest link to https://bitcoinops.org/en/newsletters/2018/12/28/#cve-2017-12842 (and move link to references at the bottom)